docs/reg_service_guide.md

```diff
@@ -1,82 +1,94 @@
-# Registering ZMS Service Identity
-----------------------------------
```

Comment:
Replaced the content of this doc with the content from our internal documentation (using UI to generate keys instead of zms-cli)